### ISSUE TITLE
Case sensitive autopacking

### ISSUE THREAD

**Issue by MAGNAT2645** [ruarai/CompilePal#73]
Autopack always pack files with lowercase filenames. Would be good if this feature could pack files with same filenames as you have.

**Comment by maxdup** [comment]
Yes, source is not case sensitive and CompilePal takes advantage of that by lowercasing everything to make searching easier. I hadn't thought about this before but you bring a good point. I'd be tempted to output paths as they appear in the bsp instead, so that everything is consistent in the output bsp. But I think your idea is better. It's better to not have multiple versions of the same assets under different filenames floating around.

**Comment by Exactol** [comment]
After some quick investigation, it looks like we do output everything as appears in the BSP, and bspzip is the one lower casing the filenames @maxdup . As far as I'm aware, Source isn't case sensitive, so is there any reason you want this feature @MAGNAT2645 ?

**Comment by MAGNAT2645** [comment]
I wanted the case sensitivity to work like in VIDE Pakfile. Sometimes mappers leave Easter eggs in file names or just leave funny names. Case-sensitive text is more readable.

**Comment by Exactol** [comment]
I looked into this, but I can't figure out how VIDE managed to pack it with uppercase. I'm putting this feature on hold until I can find out how.
